### PR TITLE
Project Music Releases Johanna R & Johanna Å

### PR DESCRIPTION
**Pull request by jastrand**
Link to my deployed project: 
https://infallible-sammet-2dbca6.netlify.com/

My reflections on how this week's project turned out:
We manage to start up the project early and learned by discussing our different issues. We are happy with how this project turned out. 

Things I'd like to have clarified or explained in more detail:
If the album name is long, the artist names are printed at different levels in the desktop view. We tried many techniques to solve this, but did not manage in the end. We would like some advice on this. We also wonder if css "fill: white" is only possible when converting svg files to components? We only managed to make them white with another solution using filter: invert.

### CODE REVIEW

File: code/src/Album.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  width: 300px;
+  height: 300px;
+  display: flex;
+  justify-content: space-evenly;
+  position: absolute;
+  background: black;
+  opacity: 0.75;
+}
+
+.Img-wrapper {
```
Comment (ericaglimsholt) [inline code/src/Album.css]: How come these to starts with big letters when the rest is starting with a small letter? :)

File: code/src/Albums.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Artist } from './Artists'
+import { HoverIcons } from 'Hovericons'
+
+
+// This compontent will fetch all data related to the Album including Album name and album image.
```
Comment (ericaglimsholt) [inline code/src/Albums.js]: Nice comment! Very useful

File: code/src/Albums.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+export const Album = (props) => {
+  return (
+    <article className='album-card'>
+      <div className="Img-wrapper">
```
Comment (ericaglimsholt) [inline code/src/Albums.js]: You seem to use `'` everywhere else so it's good if you follow the same here (and check my comment before this one about the big letter)
```suggestion
      <div className='img-wrapper'>
```

File: code/src/Albums.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        <HoverIcons />
+        <img src={props.item.images[1].url} alt='album cover' />
```
Comment (ericaglimsholt) [inline code/src/Albums.js]: Suggestion: having the album title as `alt`

File: code/src/Hovericons.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (ericaglimsholt) [inline code/src/Hovericons.css]: This file needs some work on the indention

File: code/src/Album.css (hunk 1 above)
Comment (jastrand) [reply code/src/Album.css]: I believe it was a rookie mistake on this one - would you say that small letters are best to work with using when declaring classes?

File: code/src/Albums.js (hunk 4 above)
Comment (jastrand) [reply code/src/Albums.js]: Great idea!

**Pull request closed without merging**